examples/diffusion_limited_aggregation.jl

```diff
@@ -0,0 +1,123 @@
+using Agents, Random, LinearAlgebra
```

Comment:
So the next step will be to write up some documentation for this. You've seen the `examples` section already, have go at doing something similar, I'll add some comments about flow here.